lib/vsc/install/ci.py

```diff
@@ -146,8 +146,8 @@ def gen_tox_ini():
     envs = []
 
     # always run tests with Python 3
-    py3_env = 'py36'
-    envs.append(py3_env)
+    py3_envs = ['py36', 'py39']
```

Comment:
how do we run tests without 39 in certain repos?

Reply:
We can add it to `vsc-cfg.ini` to optionally disable it?

Reply:
might be a good idea, same way we did it for the py3 move.

Reply:
I'll look into it

Reply:
yes, should be dealt with same as in py3 move: first it's opt in, then mandatory with optout

Reply:
OK, I'll get something working